managed/src/main/java/com/yugabyte/yw/common/Util.java

```diff
@@ -323,4 +326,20 @@ public static String getFileChecksum(String file) throws IOException, NoSuchAlgo
   public static void moveFile(Path source, Path destination) throws IOException {
     Files.move(source, destination, REPLACE_EXISTING);
   }
+
+  public static ArrayNode getUniverseDetails(Set<Universe> universes) {
```

Comment:
Can you quickly tell me why these are the fields that are required, or what the rationale behind it is?

Reply:
UI doesn't need all the fields when we click on show universes in certificate or KMS provider or backup configs.
If we pass the entire json, UI has to iterate through JSON and get this info. If the back end provides this info directly, then UI can directly utilize it.